src/tables.js

```diff
@@ -11,6 +11,7 @@ const Flow = {
   STEM_WIDTH: 1.5,
   STEM_HEIGHT: 35,
   STAVE_LINE_THICKNESS: 1,
+  STAVE_LEDGER_LINE_THICKNESS: 2,  // Gould, Behind Bars: "about twice as thick"
```

Comment:
What's `STAVE_LINE_THICKNESS` used for? (I thought I had used that for ledger line thickness.)

Reply:
STAVE_LINE_THICKNESS is the default thickness of a normal staff line.  

I've renamed STAVE_LEDGER_LINE_THICKNESS to LEDGER_LINE_THICKNESS_MULTIPLIER and now use as a default STAVE_LINE_THICKNESS * LEDGER_LINE_THICKNESS_MULTIPLIER